### PR TITLE
performance: increase perfomance when build with -tag release

### PR DESCRIPTION
**Pull request by pnsafonov**
Conditional compilation to fix [https://github.com/wader/fq/issues/408](https://github.com/wader/fq/issues/408)

### CODE REVIEW

File: pkg/build/norelease.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//go:build !release
+
+package build
+
+func initReleaseOptions() {
+	CheckChildAlreadyExists = true
+}
```
Comment (wader) [inline pkg/build/norelease.go:7]: I think you can skip option.go and have this in norelease.go
```
const CheckChildAlreadyExists = true
```
and this in release.go
```
const CheckChildAlreadyExists = false
```

**Comment by wader** [comment]
But did my patch earlier to use a map fix the performance issue? if so i would probably prefer that to keep things simple, generally try to avoid having options if possible

**Comment by pnsafonov** [comment]
> But did my patch earlier to use a map fix the performance issue? if so i would probably prefer that to keep things simple, generally try to avoid having options if possible

 Yes, patch with map fix performance issue. Same performance as conditional compile approach.
 Merge request:
 [https://github.com/wader/fq/pull/411](https://github.com/wader/fq/pull/411)

**Comment by pnsafonov** [comment]
Alternative is used:
[https://github.com/wader/fq/pull/411](https://github.com/wader/fq/pull/411)

**Pull request closed without merging**